modules/tls-profiles-ingress-configuring.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-ingress-configuring_{context}"]
+= Configuring the TLS security profile for the Ingress Controller
+
+To configure a TLS security profile for an Ingress Controller, edit the `IngressController` custom resource (CR) to specify a predefined or custom TLS security profile. If a TLS security profile is not configured, the default value is based on the TLS security profile set for the API server.
+
+.Sample `IngressController` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
```

Comment:
@snarayan-redhat, this is the IngressController API in the `operator.openshift.io` API group:

```
oc explain ingresscontrollers --api-version=operator.openshift.io/v1 --recursive
KIND:     IngressController
VERSION:  operator.openshift.io/v1
...
FIELDS:
   apiVersion	<string>
   kind	<string>
   spec	<Object>
      tlsSecurityProfile	<Object>
         custom	<>
         intermediate	<>
         modern	<>
         old	<>
         type	<string>
         minTLSVersion	<string>
```